src/strings.cpp

```diff
@@ -95,6 +95,66 @@ const StringParameter &StringParameters::GetNextParameterReference()
 	return param;
 }
 
+/**
+ * Encode a string with no parameters into an encoded string.
+ * The encoded string can be stored and decoded later without requiring parameters to be stored separately.
```

Comment:
This line caught me off guard given the previous on talks abou there being no parameters. I guess it's fine to tell users what the idea of an EncodedString is. Either with or without is fine by me.

Reply:
Elected to remove from this comment, it remains in the other one which does take parameters.